dev-reverse-proxy/run-caddy.sh

```diff
@@ -6,5 +6,11 @@
 #
 
 set -e
-curl -o caddy "https://caddyserver.com/api/download?os=linux&arch=amd64"
+
+if [ ! -f caddy ]
+then
+curl -o caddy "https://caddyserver.com/api/download"
 chmod +x ./caddy
```

Comment:
Should we make this usable for Windows or macOS, too?

Reply:
already fixed that